### PR TITLE
Remove orphan condor jobs

### PR DESCRIPTION
**Pull request by mira-miracoli**
This scripts works similar like @bgruening 's [command](https://github.com/usegalaxy-eu/operations/blob/main/jobs.md#find-jobs-known-to-htcondor-that-are-not-known-to-galaxy-anymore), but also catches ITs

### CODE REVIEW

File: roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    owner: root
+    group: root
+
+- name: Add to cron
+  ansible.builtin.cron:
+    name: "Remove jobs that are running in condor but unknown to galaxy"
+    day: "*"
+    hour: "*/1"
+    minute: "0"
+    user: "{{ galaxy_user.name }}"
```
Comment (sanjaysrikakulam) [inline roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml:50]: There might be a permission issue. The `galaxy` user runs the Cron job, while the `root` user owns the log directory.

```
- name: Create cron log directory
  ansible.builtin.file:
    state: directory
    path: "{{ remove_orphan_jobs_log }}"
    mode: 0755
    owner: root
    group: root
```

Comment (mira-miracoli) [reply roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml:50]: I think `0755` should equal to `-rw-rw-rw-`

Comment (sanjaysrikakulam) [reply roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml:50]: `0755`  is `rwxr-xr-x`

Comment (mira-miracoli) [reply roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml:50]: yes that is true, but it is only the directory, the file itself has `644`and is owned by galaxy

Comment (sanjaysrikakulam) [reply roles/usegalaxy-eu.remove-orphan-condor-jobs/tasks/main.yml:50]: OK.

**Pull request merged**